spec/system/checkout_spec.rb

```diff
@@ -611,6 +617,7 @@
 
       it "displays the entered state name without evaluating" do
```

Comment:
This example seems fails frequently. See for example, https://app.circleci.com/pipelines/github/nebulab/solidus_starter_frontend/879/workflows/778f9ce5-62d3-43b4-af6b-6e282b275815/jobs/2243/tests. In the test run I did in the video below, I got it to fail 40 out of 100 times.

I discovered that the example fails frequently under the following conditions:

1. When signing in as a guest during checkout.
2. When using the apparition JS driver.

However, under ANY of these conditions, the example succeeds most of the time:

1. On the current master branch, where we don't have authentication yet.
2. When signing in as a user during checkout.
3. When using a different JS driver, like selenium_chrome_headless .

Please see the video below for more details. 

https://www.loom.com/share/2a6db47101f24fcd8206361f8300d7f9

For the meantime, I'll add a commit to apply the workaround I did in the video, i.e. sign in as user instead of gueset.

Reply:
Pushed commit https://github.com/nebulab/solidus_starter_frontend/pull/172/commits/effe34d81bd7ee07a9dc34ce5e990e9331667727.